src/pages/guides/index.md

```diff
@@ -286,22 +286,50 @@ You can also consider implementing a retry mechanism to call public key urls in
 
 **Verifying the Signature**
 
-Once you have the public keys fetched as plain text, you can now verify the digital signatures by following the steps as below:
+Once you have the pem public keys, you can now verify the digital signatures by following the steps as below:
 
-1. Decrypt the message digest using the public key.
-2. Compute the hash message digest of the event using `rsa-sha256` hash function algorithm. For computing:
-   - Use the webhook request payload for event deliveries.
-   - Use the challenge code in the query param for challenge deliveries.
-3. Validate each signature by comparing:
-   - the digest received (in step 1) after decrypting the signature using the public key.
-   - and the message digest computed (in step 2) by hashing.
-4. If any one of the signature validation is successful, then the event is valid.
+1. Create the PublicKey object using the pem public key.
+2. Create a `Signature` (for java apps) or `crypto` -> [Verify](https://nodejs.org/docs/latest-v14.x/api/crypto.html#crypto_class_verify) (for nodeJS apps) instance using the `rsa-sha256` hashing algorithm.
+3. Supply the instance raw event payload.
+4. Use the public key and decoded signature to verify. 
+5. Do the above for both the signatures and if any one of the signature validations is successful, then the event is valid.
 
 A pictorial block diagram for the signature validation steps above that you should follow 
 
 ![Digital Signature Validation Steps](./img/digi_signature_verification_block_diagram.png "Digital Signature Validation Steps")
 
-Refer to [this](https://github.com/adobe/aio-lib-events/blob/1.1.2/src/index.js#L516) signature verification method of the events sdk to understand the above signature validation steps for your webhook app.
+Refer to [this](https://github.com/adobe/aio-lib-events/blob/1.1.5/src/index.js#L519) signature verification method of the events sdk (**nodeJS** based) to understand the above signature validation steps for your webhook app.
+
+For Java based webhook applications, one can verify signature using the below code snippet.
+```java
```

Comment:
For some reason, the syntax highlighting is not working for `java` code snippets. Worth finding out on #adobeio-devsite channel.

Reply:
https://adobeio.slack.com/archives/C01GU3V8XE0/p1650651774934509